Softeq.XToolkit.HttpClient/Network/HttpRequestScheduler.cs

```diff
@@ -367,17 +368,15 @@ private async Task ExecuteAsync(CompleteHttpRequestScheduledTask task)
                     }
                 }
             }
-            catch (TaskCanceledException)
-            {
-                task.Response = new HttpResponse
-                {
-                    StatusCode = HttpStatusCode.RequestTimeout,
-                    IsSuccessful = false
-                };
-            }
             catch (Exception ex)
             {
-                task.Response = _httpClientErrorHandler.FromException(ex);
+                task.Response = timeoutCancellationToken.IsCancellationRequested
```

Comment:
Could you please add a comment here about the reason? (not obvious solution for other people)